pkg/migrations/sql/versions/dev/0.7.2-dev/2-event_name.sql

```diff
@@ -0,0 +1,114 @@
+/*
+    Remove the event_name_check check constraint on SCHEMA_TRACING.event if it exists
+    You can't drop the constraint if compression is turned on for the table.
+    If compression is on, turn it off.
+
+*/
+DO $block$
+DECLARE
+    _is_timescaledb_installed boolean = false;
+    _is_timescaledb_oss boolean = true;
+    _timescaledb_version_text text;
+    _timescaledb_major_version int;
+    _timescaledb_minor_version int;
+    _is_compression_available boolean = false;
+    _compression_enabled boolean = false;
+BEGIN
+
+    -- determine whether the constraint even exists
+    -- if it doesn't, bail early
+    IF
+    (
+        SELECT count(*) = 0
+        FROM pg_constraint c
+        INNER JOIN pg_class t on (c.conrelid = t.oid)
+        INNER JOIN pg_namespace n on (c.connamespace = n.oid and t.relnamespace = n.oid)
+        WHERE n.nspname = 'SCHEMA_TRACING'
+        AND t.relname = 'event'
+        AND c.conname = 'event_name_check'
+        AND c.contype = 'c'
+    )
+    THEN
+        RETURN;
+    END IF;
+
+    /*
+        These functions do not exist until the
+        idempotent scripts are executed, so we have
+        to deal with it "manually"
```

Comment:
Is this true? If we are migrating then the idempotent scripts were run on installation right?

Reply:
i think you may be right. They may not be available in the preinstall scripts, but as you say I think they should be here for migrations.